### PR TITLE
fix: make adhoc table fit width

### PR DESCRIPTION
**Pull request by dogfrogfog**
## Brief
- https://github.com/pyroscope-io/pyroscope/issues/1581

## Changes
- make adhoc table fit width
- change adhoc table time format to _MMM d, yyyy_ at _H:mm_ 
- add "tooltip" (`title` span attribute) to Filename cells
- ~~add suffix column, the `suffix` value should be part of `/api/adhoc/v1/profiles` reponse~~

Screenshot 2022-10-05 at 12 28 33
Screenshot 2022-10-05 at 12 25 59

## Concerns
- ~~FileNames column values are hidden when  screen width is small (thats not a problem since we are adding suffix column?)~~
Screenshot 2022-10-05 at 12 27 16

### CODE REVIEW

File: webapp/javascript/components/FileList.tsx [hunk 1]
```diff
@@ -33,13 +34,13 @@ const getBodyRows = (
...
-              {profile.name}
+            <div className={styles.profileName}>
+              <span title={profile.name}>{profile.name}</span>
               {isRowSelected && <CheckIcon className={styles.checkIcon} />}
-            </>
+            </div>
           ),
         },
-        { value: profile.updatedAt },
+        { value: format(parseISO(profile.updatedAt), 'yyyy-MM-dd') },
```
Comment (eh-am) [inline webapp/javascript/components/FileList.tsx:43]: @Rperry2174 Not sure I understand the reasoning behind this. What if I am uploading multiple files (via `adhoc` command for example) in a short interval (minutes) and I am basing off that info to navigate through my files.

**Review by eh-am — COMMENTED** [review]
I like having the names fully visible, specially if you have a bunch of profiles (`.cpu`, `.mutex`, `.inuse_objects`) that would be all trimmed to `rideshare-app-golang...`, but from design purposes I guess it makes sense making it "prettier".

File: webapp/javascript/components/FileList.tsx (hunk 1 above)
Comment (Rperry2174) [reply webapp/javascript/components/FileList.tsx:43]: sounds good for time lets do this format:
image

Lets add one more column with the suffix:
`cpu`, `mutex`, `inuse_objects`, etc

**Comment by dogfrogfog** [comment]
updated PR description

**Comment by dogfrogfog** [comment]
https://user-images.githubusercontent.com/47758224/194075862-e798ed9b-631f-4252-819c-b1a73e854d83.mov

File: webapp/javascript/components/FileList.tsx [hunk 2]
```diff
@@ -29,17 +30,23 @@ const getBodyRows = (
       (profId) => profId === profile.id
     );
 
+    const date = parseISO(profile.updatedAt);
+    const timeString = `${format(date, 'MMM d, yyyy')} at ${format(
+      date,
+      'H:mm'
+    )}`;
```
Comment (eh-am) [inline webapp/javascript/components/FileList.tsx:37]: I think we are missing `PM/AM` there
image

Comment (eh-am) [reply webapp/javascript/components/FileList.tsx]: Yeah, I like the 24h but it's definitely confusing, specially when it's alone like this. Plus we can say we have a large US audience.

Comment (Rperry2174) [reply webapp/javascript/components/FileList.tsx]: I like 24h format I think this is fine

**Comment by eh-am** [comment]
Closes https://github.com/pyroscope-io/pyroscope/issues/1338

**Review by Rperry2174 — APPROVED** [review]
lgtm

**Pull request merged**